versions/1.0/index.md

```diff
@@ -26,7 +26,7 @@ Servers **MUST** send all JSON data in response documents with the header
 
 Anywhere a **URI** is specified, it must adhere to the following rules:
 
-* **MUST** be a valid URI.
+* **MUST** be a valid URI that is active and responds to requests.
```

Comment:
How about `Must be a valid URI that is active and returns a resource`? Also should we add language around not having empty paths within the URI that are solely used for grouping?

Reply:
@teehill actually can we revert back to how you had it originally? I think your wording is more generic and covers situations where an endpoint does not return a resource. (ie POST only calls)

Reply:
done